### ISSUE TITLE
Compiling without Webpack

### ISSUE THREAD

**Issue by tmattio** [jchavarri/jsoo-react#47]
Hi @jchavarri!

I've been setting up `jsoo-react` in my project and thought I'd give some feedback on the setup 🙂 

To give you some context, I'm using `ocaml-crunch` to generate OCaml modules for my static content. It simplifies the deployment drastically: with this setup, I can just ship an executable and don't have to worry about storing static files.
To serve the React app, I am embedding the compiled Js in an OCaml module and serve it from one of the endpoints of the server.

Now, all of this is pretty much automated with Dune configurations, however, the React app first needs to be processed by Webpack (or another bundler), because the code generated by Js_of_ocaml imports `react`.

This dependency on Webpack adds a lot of overhead to the setup, and I'm wondering if it would be possible to get rid of the imports in the generated Js code.

The HTML page could potentially get the react library from their CDN:

```
<script crossorigin src="https://unpkg.com/react@16/umd/react.production.min.js"></script>
<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.production.min.js"></script>
```

And the compiled Js could just assume that `React` is available.

I'm not very familiar with Js_of_ocaml, so I have no idea if that's feasible, but if it is, it would have some nice benefits:

- Simplify the setup by removing the dependency on Webpack.
- Improve build times, because we wouldn't need Dune to check the `node_modules`.
- (Maybe?) this would simplify Server Side Rendering if one-day `jsoo-react` uses `Tyxml`

What do you think?

**Comment by jchavarri** [comment]
I think it's a great idea! I wouldn't like to remove Webpack support, because it might be interesting in the future (for things like lazy loading) and I think some folks will appreciate that jsoo-react has some guidelines on how to be used with Webpack.

But otherwise, I love the simplification and build times improvements :) Your proposal made me realize we have no good reason to assume anything about how React gets provided to final bundles, we can just do nothing inside the library and resolve the globals in userland. I created a PR in #48, let me know what you think!

**Comment by jchavarri** [comment]
Fixed in #48.
